### PR TITLE
[docs] Add more UPK key descriptions

### PR DESCRIPTION
**Pull request by szupi-ipuzs**
Based on description from tuya website (links provided in #218) I have filled in some of the missing information. The description was for bluetooth devices, but the keys I described seem to be more or less universal.

### CODE REVIEW

File: docs/resources/tuya-pin-config.md [hunk 1]
```diff
@@ -113,9 +119,23 @@ Key(s)                                      | Meaning
 `keyccfg1`, `keyccfg2`                      |                                       |
 `keyfunc`, `keyglobefunc`                   |                                       |
 `keylt`, `keynumber`                        |                                       |
+**Pairing-related**                         |                                       |
```
Comment (szupi-ipuzs) [inline docs/resources/tuya-pin-config.md:122]: I don't really see a need for keeping detailed description of the pairing-related keys.
I doubt anyone would want to duplicate this behavior in esphome :)

IMHO, simply enumerating them as ones should be sufficient.

Comment (kuba2k2) [reply docs/resources/tuya-pin-config.md:122]: I wanted to keep the "known, not relevant" keys from the "unknown, not described yet" keys.

File: docs/resources/tuya-pin-config.md [hunk 2]
```diff
@@ -24,27 +27,30 @@ Key(s)                                      | Meaning
 `iicsda`                                    | I²C SDA Pin                           |
 `net_trig`                                  |                                       |
 `net_type`                                  |                                       |
-`wfcfg`                                     | Pairing related - not relevant        | `spcl` / `spcl_auto` / `prod`
 `wfct`                                      |                                       |
 **Lights/bulbs**                            |                                       |
 `cmod`                                      | Color Mode                            | `rgbcw` / `rgb` / `cw` / `c` / `rgbc`
+`dmod`                                      | Light driver type                     | 0 - PWM<br>1 - SM16726B<br>2 - SM2135E<br>3 - SM2135EH<br>4 - SM2135EJ<br>5 - BP1658CJ
```
Comment (Cossid) [inline docs/resources/tuya-pin-config.md:33]: 6 is `BP5758D` and 7 is `SM2235/SM2335`

Comment (kuba2k2) [reply docs/resources/tuya-pin-config.md:33]:
```suggestion
`dmod`                                      | Light driver type                     | 0 - PWM<br>1 - SM16726B<br>2 - SM2135E<br>3 - SM2135EH<br>4 - SM2135EJ<br>5 - BP1658CJ<br>6 - BP5758D<br>7 - SM2235/2335
```

File: docs/resources/tuya-pin-config.md [hunk 3]
```diff
@@ -2,14 +2,21 @@
...
 Key(s)                                      | Meaning                               | Possible values
---------------------------------------------|---------------------------------------|---------------------------------------------------------
-`crc`                                       |                                       |
-`module`                                    |                                       |
-`category`                                  |                                       |
-`Jsonver`<br>`jv`                           |                                       |
+--------------------------------------------|---------------------------------------|---------------------------------------------------------------------------------------
+`crc`                                       | UPK data checksum                     |
+`module`                                    | Tuya module used                      | `CB3S` / `WB3S` / `CBU`, etc.
+`category`                                  | Device type as a number               | 0502 - CW light<br>0505 - RGBCW light
```
Comment (Cossid) [inline docs/resources/tuya-pin-config.md:18]: This actually belongs in the `Lights/bulbs` section, it only exists for bulbs, and they always start with `050`
The last number is number of channels
`0501` is monochromatic (usually warm white)
`0503` is RGB
`0504` is RGBW
but all this is covered by `cmod` anyway

File: docs/resources/tuya-pin-config.md [hunk 4]
```diff
@@ -24,27 +27,30 @@ Key(s)                                      | Meaning
...
+`dmod`                                      | Light driver type                     | 0 - PWM<br>1 - SM16726B<br>2 - SM2135E<br>3 - SM2135EH<br>4 - SM2135EJ<br>5 - BP1658CJ
 `cwtype`                                    | Color temperature driver              | 0 - cool and warm white (CW)<br>1 - correlated color temperature (CCT)
-`brightmin`, `brightmax`                    | Min/Max Brightness                    | 0%-100%
-`cwmin`, `cwmax`                            | Cold-Warm Min/Max Brightness          | 0%-100%
+`onoffmode`                                 | On/off gradient enabled               | 0 / 1
+`pmemory`                                   | Power-off memory enabled              | 0 / 1
+`defcolor`                                  | Default Color                         | `c` / `r`
+`defbright`                                 | Default Brightness                    | 0%-100%
+`deftemp`                                   | Default Color Temperature             | 0-100 when defcolor is cool white
 `cwmaxp`                                    | Cold-Warm Max Power                   | 0%-100%
```
Comment (szupi-ipuzs) [inline docs/resources/tuya-pin-config.md:40]: The values here are wrong, the docs say it's 100-200 with a pitch of 10. I've actually just flashed a lamp which has it set to 200.

Comment (Cossid) [reply docs/resources/tuya-pin-config.md:40]: Please provide an example.  CloudCutter has ~175 UPKs with `defbright` and `deftemp` and none are above 100.

Comment (szupi-ipuzs) [reply docs/resources/tuya-pin-config.md:40]: No problem, I should've been more precise, but github's UI is still new to me (i'm used to gerrit).
@kuba2k2 Sure, this note was more for me to remember later :)

File: docs/resources/tuya-pin-config.md [hunk 5]
```diff
@@ -111,6 +111,12 @@ Key(s)                                      | Meaning
 `pirsense_pin` + `pirsense_lv`              | PIR sensitivity (PWM) + Active Level  |
 `pirrange`                                  |                                       |
 `pirwarn`                                   |                                       |
+**Ambient light sensor**                    |                                       |
```
Comment (szupi-ipuzs) [inline docs/resources/tuya-pin-config.md:114]: These are actually a guess, but I have 4 lamps (2 brennenstuhls and 2 lsc) with PIR and ambient light sensor and can observe that these valuse more or less match the actual lighting. And they do differ between models a bit.

**Review by kuba2k2 — APPROVED** [review]

**Pull request merged**